apps/greenhouse-management/src/components/OrgInfo.jsx

```diff
@@ -1,23 +1,55 @@
-import React, { useMemo } from "react"
-import { Stack } from "juno-ui-components"
+import React, { useMemo, useEffect, useState } from "react"
+import { createClient } from "sapcc-k8sclient"
+import { useApiEndpoint, useAuthData } from "./StoreProvider"
+
+// Shows organization info
 
 const OrgInfo = () => {
-  // temporay fix to display org name until we have the k8sclient to fetch the metadata
-  const orgName = useMemo(() => {
-    const currentUrl = new URL(window.location.href)
-    let match = currentUrl.host.match(/^(.+)\.dashboard\..+/)
-    return match ? match[1] : currentUrl.searchParams.get("org")
-  }, [])
+  const apiEndpoint = useApiEndpoint()
+  const [org, setOrg] = useState(null)
+  console.log(apiEndpoint, "apiEndpoint")
+
+  const authData = useAuthData()
+
+  const namespace = useMemo(() => {
```

Comment:
would you mind to rename this var to orgName. It is not meant to be as namespace in this case